initfiles/bash/etc/init.d/hpcc_common.in

```diff
@@ -445,6 +435,30 @@ startCmd() {
       fi
     fi
 
+    limits=(
+      MIN_Hn_nofile="32768"
+      MIN_Hc_core="unlimited"
+      MIN_Hu_nproc="8192"
+      MIN_Hr_rtprio="4"
+      MIN_Hl_memlock="unlimited"
+      MIN_Hs_stack="512" )
```

Comment:
Well for example.  On my little laptop right now its sitting at 8192 for the stack size (I assume each thread would be allowed this as well.).  So in my mind, 512 probably my won't get set at all, as that's a default value which basically means the available memory on the system is exceptionally low.